commands/info.js

```diff
@@ -56,7 +61,7 @@ function * run (context, heroku) {
         path: `/client/v11/databases/${addon.name}`
       }).catch(err => {
         if (err.statusCode !== 404) throw err
-        cli.warn(`${cli.color.addon(addon.name)} is not yet provisioned.\nRun ${cli.color.cmd('heroku pg:wait')} to wait until the db is provisioned.`)
+        cli.warn(`${cli.color.addon(addon.name)} is not yet provisioned.\nRun ${cli.color.cmd('heroku addons:wait')} to wait until the db is provisioned.`)
```

Comment:
Is `pg:wait` obsolete with `addons:wait`? Should we flag it as deprecated and redirect users?